app/routes/auth.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const userController = require("../controllers/userController");
+const authMiddleware = require("../middleware/authMiddleware");
+
+router.post("/signup", userController.signup);
+router.post("/login", userController.login);
+router.get("/user/profile", authMiddleware, userController.getUserProfile);
+router.put("/user/profile", authMiddleware, userController.updateUserProfile);
```

Comment:
this will be like -> `auth/user/profile`.
Think of some other route as auth works good with signup and login but not with getting users related stuff.